### ISSUE TITLE
Parameter 2 to qtranxf_postsFilter() expected to be a reference

### ISSUE THREAD

**Issue by mikaelz** [qTranslate-Team/qtranslate-x#544]
With WP version: 4.8.1
qTranslate-X version: 3.4.6.8
PHP 7.1.9
I get
`Warning: Parameter 2 to qtranxf_postsFilter() expected to be a reference, value given in /srv/http/slov-aqua.eu/wp-includes/class-wp-hook.php on line 298`

Relates #542

**Comment by kostyanius** [comment]
Please advice from where this param should be removed? What is an exact file location. Can find it.

**Comment by jomko** [comment]
@kostyanius  https://wordpress.org/support/topic/problem-with-php-7-1-2/#post-8908632

**Comment by emran-shaikh** [comment]
@jomko worked like a charm dude thanks.

**Comment by wywarren** [comment]
@jomko thanks for the solution. Is there any reason to keep the & on the query on these lines? Is it possible to commit this to the master for future compatibility?

**Comment by rordi** [comment]
Any news on this? Is this plug-in abandoned?

**Comment by davidjbrossard** [comment]
I have the same issue / question as @rordi . Any update?

**Comment by herrvigg** [comment]
Yes the plugin qTranslate-X is abandoned. We are currently reviving it in a parallel github project called qTranslate-**XT** at the moment. Check here: https://github.com/qtranslate/qtranslate-xt
These issues are fixed in this repo.

**Comment by felipecesr** [comment]
@jomko thanks for the solution.
